init_db_scripts/initdb.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
The test dump should reflect changes to the database. So if the database changes we should update the test dump here. It would be nicer to use the testdump from the db repository. Maybe create an issue for this.